cloud: add faq about how to import specified columns, and delete outdated files (#14117)
pingcap/docs: tidb-cloud/tidb-cloud-import-local-files.md

9. After the import task is completed, you can click **Explore your data by Chat2Query** to query your imported data. For more information about how to use Chat2Qury, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md).

10. On the **Import** page, you can click **View** in the **Action** column to check the import task detail.

## FAQ

### Can I only import some specified columns on TiDB Cloud?

No. Currently, you can only import all columns of a CSV file into an existing table.

> In this case, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](/sql-statements/sql-statement-load-data.md) to specify the columns to import. For example:

Author: ```suggestion
In this case, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](https://docs.pingcap.com/tidb/stable/sql-statement-load-data) to specify the columns to import. For example:
```

```sql
CREATE TABLE `import_test` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(64) NOT NULL,

>   `address` varchar(64) NOT NUL,

Reviewer: ```suggestion
  `address` varchar(64) NOT NULL,
```

  PRIMARY KEY (`id`)
) ENGINE=InnoDB;

> LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);

Reviewer: will report an error

```sql
mysql> CREATE TABLE `import_test` (
    ->   `id` int(11) NOT NULL AUTO_INCREMENT,
    ->   `name` varchar(64) NOT NULL,
    ->   `address` varchar(64) NOT NULL,
    ->   PRIMARY KEY (`id`)
    -> ) ENGINE=InnoDB;
Query OK, 0 rows affected (0.47 sec)

mysql> LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.
mysql>
```

Reviewer: I am asking serverless team

Reviewer: The problem is according to MySQL Client

> mysql  Ver 14.14 Distrib 5.7.40, for Linux (x86_64) using  EditLine wrapper
success
> mysql  Ver 8.0.26 for macos11 on arm64 (MySQL Community Server - GPL)
failed

If user encounter the failed case, they can add `--local-infile=true` when connect using `mysql`

> ```

Author: ```suggestion
```

If you use MySQL client and encounter the following error "ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.",  you can add `--local-infile=true` when connecting.
```

Reviewer: why delete ```?

Author: With ```, the format is messy. Will add it later. Ignore it.

9. After the import task is completed, you can click **Explore your data by Chat2Query** to query your imported data. For more information about how to use Chat2Qury, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md).

10. On the **Import** page, you can click **View** in the **Action** column to check the import task detail.

## FAQ

### Can I only import some specified columns on TiDB Cloud?

No. Currently, you can only import all columns of a CSV file into an existing table.

In this case, you can use [LOAD DATA](/sql-statements/sql-statement-load-data.md) to specify the columns to import. For example:

```sql
CREATE TABLE `url` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `client` smallint(5) unsigned NOT NULL,

>   `dest` varchar(5),

Author: ```suggestion
  `address` varchar(5),
```

  `employee` smallint(5) unsigned NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB;

> LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE URL FIELDS TERMINATED BY ',' (client, dest, employee);

Author: ```suggestion
LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE URL FIELDS TERMINATED BY ',' (client, address, employee);
```

> CREATE TABLE `url` (
>   `id` int(11) NOT NULL AUTO_INCREMENT,
>   `client` smallint(5) unsigned NOT NULL,
>   `address` varchar(5),
>   `employee` smallint(5) unsigned NOT NULL,
>   PRIMARY KEY (`id`)
> ) ENGINE=InnoDB;
> LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE URL FIELDS TERMINATED BY ',' (client, address, employee);

Reviewer 2: ```suggestion
CREATE TABLE `import_test` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(64) NOT NULL,
  `address` varchar(64) NOT NUL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB;
LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
```

9. After the import task is completed, you can click **Explore your data by Chat2Query** to query your imported data. For more information about how to use Chat2Qury, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md).

10. On the **Import** page, you can click **View** in the **Action** column to check the import task detail.

## FAQ

> ### Can I only import some specified columns on TiDB Cloud?
> 
> No. Currently, you can only import all columns of a CSV file into an existing table.
> 
> In this case, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](https://docs.pingcap.com/tidb/stable/sql-statement-load-data) to specify the columns to import. For example:

Author: ```suggestion
### Can I only import some specified columns into TiDB Cloud?

No. Currently, you can only import all columns of a CSV file into an existing table.

To import only some specified columns, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](https://docs.pingcap.com/tidb/stable/sql-statement-load-data) to specify the columns to import. For example:
```

> To import only some specified columns, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](https://docs.pingcap.com/tidb/stable/sql-statement-load-data) to specify the columns to import. For example:

Reviewer 3: ```suggestion
To import only some specified columns, you can use the MySQL client to connect your TiDB cluster, and then use [`LOAD DATA`](https://docs.pingcap.com/tidb/stable/sql-statement-load-data) to specify the columns to be imported. For example:
```

```sql
CREATE TABLE `import_test` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(64) NOT NULL,
  `address` varchar(64) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB;
LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
```

> If you use a MySQL client and encounter the following error "ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.", you can add `--local-infile=true` when connecting.

Author: ```suggestion
If you use `mysql` and encounter the following error "ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.", you can add `--local-infile=true` when connecting.
```

> If you use `mysql` and encounter the following error "ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.", you can add `--local-infile=true` when connecting.

Reviewer 3: ```suggestion
If you use `mysql` and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
```
